pkg/util/devices.go

```diff
@@ -146,6 +159,16 @@ func SelectDynamicDirectPathIO(
 	](devices)
 }
 
+// HasPVC returns true if the VirtualMachine spec has a Persistent Volume claim.
```

Comment:
Please remove this function. This package should not link to *any* VM Op API packages. If this function is required, move it to where it is called as an anon function.